tests/infra_test.go

```diff
@@ -71,6 +74,104 @@ var _ = Describe("Infrastructure", func() {
 		return tests.WaitForSuccessfulVMIStart(obj)
 	}
 
+	Describe("Taints and toleration", func() {
+
+		It("should tolerate CriticalAddonsOnly toleration", func() {
+
+			var kvPods *k8sv1.PodList
+			By("finding all nodes that are running kubevirt components", func() {
```

Comment:
s/nodes/pods/ ?

Reply:
no, I'm searching for the nodes (through the pods)